src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -134,7 +134,8 @@ public Response getRtFeeds() {
     public Response startMonitor(
             @PathParam("id") int id,
             @QueryParam("clientId") String clientId,
-            @DefaultValue("10") @QueryParam("updateInterval") int updateInterval) {
+            @DefaultValue("10") @QueryParam("updateInterval") int updateInterval,
+            @QueryParam("enableShapes") String enableShapesStr) {
```

Comment:
The `enableShapesStr` parameter should have a default value of `true`.

Reply:
@barbeau the default value of 'true' is set in index.js. Do you think its necessary even at this location?

Reply:
Yes, I think it's necessary here - let's assume that UIs/clients other than our `index.js` might use this API.  So, I'd like this to work without considering what exists in `index.js`.